packages/batteries/src/http/http.js

```diff
@@ -171,10 +171,14 @@ export default function registerHttpEffect(
   });
 
   function dispatchEvent(event, ...data) {
-    if (!event) {
+    if (eventIsUndefined(event)) {
```

Comment:
This change affects all http effects. I'd prefer to maintain the parameter as mandatory for the htttp.get effect because
I think we shouldn't be able to use a http.get effect and ignore the response.
That's why I can't agree with this particular implementation.
Sorry...

Reply:
What should be the expected behaviour? Throw an exception saying that this is not a valid event?

Reply:
I think, this last option Maybe could be a good middle groud. It would be a similar behaviour as we do with an event dispatched not having any handler. What do you think?

Reply:
**It could be an option to throw an exception in this case, and I agree with that if we finally go with it.** But, we must keep in mind that with this approach, we put the constraint to implement a dummy success event for legacy systems that only accept GET as a valid method, even if you're doing a POST/PUT behind the scenes. (I'm thinking about Thribee & co.)

Reply:
I've just pushed a new version. Let me know what you think :)